device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
```

Comment:
model is FIXME?

Reply:
`FIXME` is a placeholder, waiting for Mellanox to provide expected value.

Reply:
@keboliu: What should be the expected value from `chassis.get_model()` on this platform?

Reply:
Per @keboliu, 'model' can vary from device-to-device. Please remove this altogether.